framework/include/base/FEProblem.h

```diff
@@ -324,13 +325,16 @@ class FEProblem :
    */
   void outputStep(ExecFlagType type);
 
+  ///@{
   /**
    * Ability to enable/disable all output calls
    *
-   * This is needed by RattleSNake/YAK to disable output because of the Yo Dawg executioners calling
+   * This is needed by Multiapps and RattleSNake/YAK to disable output because of the Yo Dawg executioners calling
```

Comment:
Do we want comments referring to internal applications, especially since this is part of the doxygen doco?